Raven.StorageExporter/StorageExporter.cs

```diff
@@ -384,6 +384,12 @@ private void WriteFilesAsAttachments(JsonTextWriter jsonWriter)
 
         private bool ShouldSkipFile(FileHeader header)
         {
+            RavenJToken rjt;
+            if (header.Metadata.TryGetValue("Raven-Delete-Marker", out rjt))
+            {
+                if (rjt.Value<bool>())
+                    return true;
+            }
```

Comment:
why not `header.IsTombstone` instead? :)

Reply:
Because i modified this before your comment...